archive/Thematic/2021/Februrary/progressed/bridgecrew2/template.html

```diff
@@ -753,8 +763,24 @@ <h1 class="null" style="text-align: center;display: block;margin: 0;padding: 0;c
 
 															<td valign="top" class="mcnTextContent" style="padding: 0px 18px 9px;line-height: 150%;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;text-align: left;">
 
-																XXXX
-
+																<p dir="ltr" style="text-align: left;line-height: 150%;margin: 10px 0;padding: 0;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;"><span style="font-size:17px"><span style="font-family:trebuchet ms,lucida grande,lucida sans unicode,lucida sans,tahoma,sans-serif"><strong id="docs-internal-guid-f6b1a81c-7fff-34dc-bbec-422bbeefc6f2"><strong style="font-weight: bolder;">Busting 5 More Myths about DevOps</strong></strong></span></span></p>
+																<br/>
+																<ol>
+																	<li><strong style="font-weight: bolder;">DevOps Doesn’t Work for Enterprises</strong></li>
```

Comment:
I think our generator makin errors.
sub-headings were wrapped to ol->li

i'll accept it, but we need to update it and replace with h2 or h3.

Reply:
I'll just update those Headings. 👍

Reply:
use this https://github.com/atherdon/newsletters/tree/master/archive/Thematic/snippets#h2